ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -86,14 +102,16 @@ azure_connection::delete_object(const std::string &object_key) const
     std::string obj = _object_prefix + object_key;
 
     auto object_client = _azure_client.GetBlobClient(obj);
-    auto delete_blob_response = object_client.DeleteIfExists();
 
-    // Returns false if obj doesn't exist.
-    if (!delete_blob_response.Value.Deleted) {
-        std::cerr << obj + " : No such object file exists in the bucket." << std::endl;
+    // Delete will delete the blob if it exists or an exception.
```

Comment:
Similar to above:
```
// Delete will delete the blob if it exists or throw an exception on failure.
```

Reply:
resolved here https://github.com/wiredtiger/wiredtiger/pull/8750/commits/e42f214727eda92b86bcea0d45fc6c00676173e8